src/components/molecules/NewsCard.unit.js

```diff
@@ -0,0 +1,109 @@
+import NewsCard from "./NewsCard";
+
+describe("@components/NewsCard", () => {
+	it(...isValidComponent(NewsCard));
+
+	it("Renders default content", () => {
+		const wrapper = mount(NewsCard, {
+			...createComponentMocks({ router: true }),
+			propsData: {
+				id: "1",
+				category: "Bio",
+				title: "Bio Bio",
+				createdAt: "2018-08-08",
+				createdBy: "Me",
+			},
+		});
+		expect(wrapper.find(".header").isEmpty()).toBe(false);
+		expect(wrapper.find(".footer").exists()).toBe(false);
+	});
+
+	it("Renders without picture and date", () => {
+		const wrapper = mount(NewsCard, {
+			...createComponentMocks({ router: true }),
```

Comment:
The way you mock the BaseLink below with RouterLinkStub does not work.
Please use the following snippet to mock components:
```suggestion
			...createComponentMocks({
				router: true
				stubs: {
					"base-link": true,
				},
			}),
```